internal/configs/annotations.go

```diff
@@ -372,6 +389,43 @@ func parseAnnotations(ingEx *IngressEx, baseCfgParams *ConfigParams, isPlus bool
 			}
 		}
 
+	}
+	if hasAppProtectDos {
+		if appProtectDosEnable, exists, err := GetMapKeyAsBool(ingEx.Ingress.Annotations, "appprotectdos.f5.com/app-protect-dos-enable", ingEx.Ingress); exists {
```

Comment:
should we replace these string values with the const, i.e. `AppProtectDosPolicyAnnotation` ?

Reply:
Its just copy past from nap, I can change it both for nap and dos.

If u approve will make the changes for both

Reply:
Copy from nap:
if hasAppProtect {
		if appProtectEnable, exists, err := GetMapKeyAsBool(ingEx.Ingress.Annotations, "appprotect.f5.com/app-protect-enable", ingEx.Ingress); exists {

Do we want to make changes both in nap + napdos?

Reply:
yeah that would be great, thanks.

Reply:
I look on the const value all of them are used by another file and this is the reason for the const.